app/routes/send-endringsmelding.tsx

```diff
@@ -18,6 +18,6 @@ export async function action({ request }: ActionArgs) {
   });
```

Comment:
Denne filen kan slettes, kanskje ikke innenfor denne oppgaven da

Reply:
Laget et teknisk gjeld kort på dette 😎
https://trello.com/c/cMNCnxD1/142-fjerne-gammel-kode-for-gamle-oppsettet